pkg/apimachinery/registered/registered.go

```diff
@@ -99,12 +121,6 @@ func IsEnabledVersion(v unversioned.GroupVersion) bool {
 	return found
 }
 
-// IsRegisteredVersion returns if a version is registered.
-func IsRegisteredVersion(v unversioned.GroupVersion) bool {
```

Comment:
> I remember @deads2k specifically asked me to add these functions when reviewing my PR. @deads2k do you still want to keep these?

Yes, having a specific method like this makes deconflicting requested bits from available bits in something like the API Installer.  I haven't gotten a chance to go through the code where its created in the master, but I wanted them to resolve item 2 in this list: https://github.com/kubernetes/kubernetes/issues/16173